.eslintrc

```diff
@@ -76,7 +76,7 @@
         "no-shadow": 2,
         "no-undef": 2,
         "no-unused-vars": 0,
-        "no-use-before-define": 1,
+        "no-use-before-define": ["error", { "functions": false }],
```

Comment:
Hi @milafrerichs, I use this way to make the code more readable, hiding the implementation details, in the other hand, function declarations are hoisted and doesn't matter using a function before it is defined.